spreadsheets/sort_rank/landuse.csv

```diff
@@ -1,115 +1,116 @@
 kind,geometrytype,sort_rank
-power_line,*,274
-power_minor_line,*,273
-crane,*,272
-city_wall,*,271
-wall,*,270
-fence,*,269
-embankment,*,278
-snow_fence,*,267
-retaining_wall,*,266
-dam,LineString;MultiLineString,265
-tree_row,*,264
-hedge,*,263
-bridge,*,229
-pier,*,228
-dike,*,227
-cutline,*,226
-groyne,*,225
-breakwater,*,224
-dam,*,223
-land,*,222
-wetland,*,220
-mud,*,219
-footway,*,126
-tower,*,125
-wilderness_hut,*,124
-parking,*,123
-picnic_site,*,122
-water_park,*,121
-amusement_ride,*,120
-water_slide,*,119
-carousel,*,118
-attraction,*,117
-artwork,*,116
-animal,*,115
-pitch,*,114
-playground,*,113
-pedestrian,*,112
-dog_park,*,111
-garden,*,110
-beach,*,109
-hanami,*,108
-maze,*,107
-camp_site,*,106
-generator,*,105
-allotments,*,104
-plant,*,103
-summer_toboggan,*,102
-roller_coaster,*,101
-enclosure,*,100
-petting_zoo,*,99
-aviary,*,98
-theatre,*,97
-library,*,96
-cinema,*,95
-fuel,*,94
-school,*,93
-cemetery,*,92
-place_of_worship,*,91
-quarry,*,90
-railway,*,89
-farmyard,*,88
-substation,*,87
-works,*,86
-water_works,*,85
-wastewater_plant,*,84
-common,*,83
-village_green,*,82
-sports_centre,*,81
-stadium,*,80
-runway,*,79
-apron,*,78
-taxiway,*,77
-aerodrome,*,74
-airfield,*,73
-recreation_track,*,72
-caravan_site,*,71
-recreation_ground,*,70
-quay,*,69
-wharf,*,68
-port_terminal,*,67
-ferry_terminal,*,66
-container_terminal,*,65
-shipyard,*,64
-boatyard,*,63
-golf_course,*,62
-aquarium,*,61
-hospital,*,60
-trail_riding_station,*,59
-resort,*,58
-zoo,*,57
-winery,*,56
-wildlife_park,*,55
-theme_park,*,54
-fort,*,53
-prison,*,52
-range,*,51
-danger_area,*,50
-naval_base,*,49
-military,*,48
-retail,*,47
-commercial,*,46
-service_area,*,45
-rest_area,*,44
-college,*,43
-university,*,42
-farmland,*,41
-farm,*,40
-winter_sports,*,39
-scrub,*,37
-meadow,*,36
-grass,*,35
+power_line,*,275
+power_minor_line,*,274
+crane,*,273
+city_wall,*,272
+wall,*,271
+fence,*,270
+embankment,*,279
+snow_fence,*,268
+retaining_wall,*,267
+dam,LineString;MultiLineString,266
+tree_row,*,265
+hedge,*,264
+bridge,*,230
+pier,*,229
+dike,*,228
+cutline,*,227
+groyne,*,226
+breakwater,*,225
+dam,*,224
+land,*,223
+wetland,*,221
+mud,*,220
```

Comment:
mud, wetland, land and above don't need to increment their numbers, as there is a discontinuity buffer between mud and footway.

Reply:
So, should I remove `dam`, `land`, `wetland` and `mud`? Will they sort properly when they will be removed from this list?